### DISCUSSION TITLE
wagmi v2

### DISCUSSION THREAD

**Discussion by Rickshanelorie** [WalletConnect/walletconnect-monorepo#4134] [Developer Support] [answered]
when will you guys update the next js web3Modal with wagmi v2 because both dependencies without an error

**Comment by Rickshanelorie** [comment]
@glitch-txs

**Reply by glitch-txs** [reply, marked as answer]
Wagmi v2 is now supported: https://docs.walletconnect.com/web3modal/about

**Reply by Rickshanelorie** [reply]
hell yeah awesome
